### PR TITLE
Card Component

### PR DESCRIPTION
**Pull request by nibivi77**
### Description

Added Card Component.

### Additional Notes
- Tests are coming in a separate PR.

### CODE REVIEW

File: __tests__/pages/home.test.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ describe('Home page', () => {
 
   it('should render the page', () => {
     render(<Home locale="en" content={content} />)
-    const heading = screen.getByRole('heading')
+    const heading = screen.getByText(en.pageHeading.title)
```
Comment (s-laugh) [inline __tests__/pages/home.test.js:39]: This is going to have to change again when we move to the CMS; is there a more permanent fix maybe? (We don't care that the heading exists per say, but that the page rendered...)

File: pages/home.js [hunk 2]
```diff
@@ -13,6 +14,7 @@ export default function Home(props) {
     <div id="homeContent">
       <Heading id="my-dashboard-heading" title={t.pageHeading.title} />
       <p className="py-8">{props.content.paragraph}</p>
+      <Card t={t} />
```
Comment (s-laugh) [inline pages/home.js:17]: I'm conflicted over the simplicity of just passing the single prop for all text vs. specifying each one... not sure if there are other thoughts; if this is a pattern we want to continue?

File: tailwind.config.js [hunk 3]
```diff
@@ -45,6 +45,7 @@ module.exports = {
           light: '#5E8EBD',
           medium: '#173451',
           dark: '#21303F',
+          button: '#295376',
```
Comment (s-laugh) [inline tailwind.config.js:48]: Should we be adding colours outside the defined pallet, or seek correction from the design team members?

**Review by s-laugh — COMMENTED** [review]
Just posing some questions (food for thought); not blocking. 
Otherwise it looks great!

File: __tests__/pages/home.test.js (hunk 1 above)
Comment (nibivi77) [reply __tests__/pages/home.test.js:39]: Yea I was thinking about that. I would of used data-test-id by Design System does not support it. I have an idea.

File: pages/home.js (hunk 2 above)
Comment (nibivi77) [reply pages/home.js:17]: I am fine with both, I'll change it up if you prefer.

File: tailwind.config.js (hunk 3 above)
Comment (nibivi77) [reply tailwind.config.js:48]: I don't know where the pre-defined color pallet comes from. I don't mind adding colors to it. I will raise this to the stand-up tomorrow.

**Review by s-laugh — APPROVED** [review]
Great work!

**Pull request merged**